docs/moderne-cli/README.md

```diff
@@ -157,32 +240,33 @@ Now that the repository is configured, it's time to migrate it to Spring Boot 3
        ▛▀▀█▀▛▀▀▀▀▜
        ▌▟▀  ▛▀▀▀▀▜
        ▀▀▀▀▀▀▀▀▀▀▀
-   Moderne CLI v0.3.2
+    Moderne CLI 1.0.0
 
-   > Selecting repositories
-   
-   > spring-projects/spring-petclinic@2.0.0
-   Selected 1 repositories (0.13s)
-   
-   > Running recipe org.openrewrite.java.spring.boot3.UpgradeSpringBoot_3_1
-   
-   > spring-projects/spring-petclinic@2.0.0
-   Fix results at file:///Users/joanvr/workshop/spring-petclinic/.moderne/run/20230905140704-ojh83/fix.patch
-   Search results at file:///Users/joanvr/workshop/spring-petclinic/.moderne/run/20230905140704-ojh83/search.patch
-   Found results on 1 repositories (12m 55s)
-   Run mod apply . --recipe-run 20230905140704-ojh83 to apply the changes.
-   
-   MOD SUCCEEDED in (12m 56s)  
+    > Selecting repositories
+
+    > spring-projects/spring-petclinic@main
+    Selected 1 repositories
+    (0.48s)
+    > Running recipe org.openrewrite.java.spring.boot3.UpgradeSpringBoot_3_1
+
+    > spring-projects/spring-petclinic@main
+        ✅ Fix results at file:///Users/mikesol/Desktop/code/spring-petclinic/.moderne/run/20230921095147-SsGwh/fix.patch
+    Found results on 1 repositories (9m 37s)
+
+    * What to do next
+        > Click on one of the patch links above to view the changes on a particular repository
+        > Run mod study to examine data tables produced by this recipe
+        > Run npm install -g diff2html-cli to produce patch files on subsequent runs that are easier to view
+        > Run mod apply . --last-recipe-run  to apply the changes
+
+    MOD SUCCEEDED in (9m 38s)
    ```
    </details>
 
-
-4. Now you can apply the changes by running the `apply` command with the recipe-run id, 
-   or you can take a look at the generated patch file with your favourite editor. 
-   Whenever you run a recipe, you should always double-check that the changes made match your expectations:
+3. The previous command will generate a patch file (`fix.patch`) that contains the changes the recipe would make to your repository. You can examine the file with your favorite editor, or you can apply the changes to the code and use `git diff` to check out the changes. It's important to always double-check that the changes made match your expectations:
 
    ```shell
-   mod apply . --recipe-run 20230905140704-ojh83
+   mod apply . --last-recipe-run
```

Comment:
I changed `--recipe-run <>` to `--last-recipe-run` so that it's clearer and easier for people to copy and use.